templates/docs/rhpam712-kieserver-mysql.adoc

```diff
@@ -1,8 +1,8 @@
 ////
     AUTOGENERATED FILE - this file was generated via
-    https://github.com/jboss-container-images/jboss-kie-modules/blob/master/tools/gen-template-doc/gen_template_docs.py.
+    https://github.com/jboss-container-images/jboss-kie-modules/blob/main/tools/gen-template-doc/gen_template_docs.py.
```

Comment:
there's no need to update the adocs files, instead you need to make sure that the script and template files used to generate it is up to date.

Reply:
ok